Wasim-othman-week2 (added homework folder and created js exercises and project-grade-calculator)

Wasim-Oth opened the pull request.

nabudaldah commented on Week2/project-grade-calculator.js:
```diff
@@ -0,0 +1,28 @@
+'use strict'
```
Create a function here!

nabudaldah commented on Week2/recipe-card.js:
```diff
@@ -0,0 +1,9 @@
+        ingredients: ['half kilo Aubergine', '1 glas rice', 'half kilo chopped meat', '1 magi' ] 
+       };
+
+       console.log(food)
```
The challenge is to print each property of the object ... please retry!

nabudaldah commented on Week2/who-wants-drink.js:
```diff
@@ -0,0 +1,22 @@
+for (let i=0; i < rows; i++){
+    drinkTray.push(drinkTypes[i]);
+    
+    if (drinkTypes[i]== 'cola'){
+        drinkTray.push('cola');
+    }
+    if (drinkTypes[i]== 'lemonade'){
+        drinkTray.push('lemonade');
+    }
```
This logic breaks if we change parameters just slightly ... could you think of a good algorithm?

Wasim-Oth commented:
Dear Nabi, thank you for your feedback. i adjusted the homework. i commented the old codes and added the new so you can see the difference or is that not necessarily because anyway you can see the changes?
